File: Bai_104/main.py
def Try(i, sum, n):
    global cnt
    for j in range(1, 10):
        if i == 4 and sum + j == n:
            cnt += 1
        elif i < 4 and sum + j < n:
            Try(i + 1, sum + j, n)

def main():
    global cnt
    t = int(input())
    for _ in range(t):
        cnt = 0
        n = int(input())
        Try(1, 0, n)
        print(cnt)

if __name__ == "__main__":
    cnt = 0
    main()

File: Bai_104/main.cpp
#include<bits/stdc++.h>
using namespace std;

using ll = long long;
int n, cnt = 0;
void Try(int i, int sum){
    for(int j = 1; j <= 9; ++j){
        if(i == 4 and sum + j == n) ++cnt;
        else if(i < 4 and sum + j < n) 
            Try(i + 1, sum + j);
    }
}

int main(){
    ios::sync_with_stdio(false);
    cin.tie(nullptr);
    int t; cin >> t;
    while(t--){
        cnt = 0;
        cin >> n;
        Try(1,0);
        cout << cnt << endl;
    }
    return 0;
}